.github/workflows/actions.yml

```diff
@@ -62,6 +62,16 @@ jobs:
           flags: keras.applications,keras.applications-${{ matrix.backend }}
           files: apps-coverage.xml
           fail_ci_if_error: true
+      - name: Test integrations
+        run: |
+          python integration_tests/distribute_training_test.py
```

Comment:
This should be renamed `tf_distributed_training_test.py`. Also, because this is TF only, this should be conditioned to the TF backend